bahmni-emr/build.gradle

```diff
@@ -1,26 +1,113 @@
 plugins {
     id "nebula.ospackage" version "3.4.0"
+    id "java"
+}
+
+repositories {
+    mavenCentral()
+    jcenter()
+    maven {
+        url 'http://mavenrepo.openmrs.org/nexus/content/repositories/public'
+    }
+    maven {
+        url 'https://oss.sonatype.org/content/repositories'
+    }
 }
 
 group 'org.bahmni.emr'
 version project.bahmniRelease
 
 task extractOmods(type: Copy) {
-    from zipTree(file("${projectDir}/resources/distro-0.91-SNAPSHOT-distro.zip"))
+    from zipTree(file("${projectDir}/resources/distro-"+project.bahmniRelease+"-SNAPSHOT-distro.zip"))
     into file("${buildDir}/distro/")
 }
 
+task extractWar(dependsOn: 'downloadMRSWar', type: Copy) {
+    from zipTree(file("${buildDir}/resources/main/openmrs.war"))
+    into file("${buildDir}/openmrs")
+}
+
+task normalizeJarName(type: Copy) {
+    from "${buildDir}" + "/libs/"
+    into "${buildDir}" + "/libs/"
+    include(project.name + '-' + project.bahmniRelease + '.jar')
+    rename(project.name + '-' + project.bahmniRelease + '.jar', 'openmrs.jar')
+}
+
 ospackage {
     packageName = 'bahmni-emr'
     release = System.getenv('GO_PIPELINE_COUNTER') ?: "1"
     arch = NOARCH
     os = LINUX
-    user = 'bahmni'
+    user = 'root'
```

Comment:
@shilpa-iplit any reason why this is changing?

Reply:
I tested with user as 'bahmni' it was failing due to permissions issue and succeeded only after user as 'root'. As the contents are moved from openmrs to bahmni-emr, there in openmrs user was root.
@SwathiVarkala

Reply:
Reverted the changes. Please check the latest commit.